app/grandchallenge/challenges/admin.py

```diff
@@ -132,6 +138,111 @@ def send_status_update_email(self, request, queryset):
             )
 
 
+class OnTimeFilter(admin.SimpleListFilter):
```

Comment:
Inverted the overdue. Mainly from the rational that if things are good in the admin, things should be green.

The overdue annotation will likely move to a model manager in a later PR. So keeping that as the overdue.